SportsResults.TwilightSaw/Senders/EmailSender.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Extensions.Hosting;
+using SportsResults.TwilightSaw.Builders;
+using SportsResults.TwilightSaw.Controllers;
+
+namespace SportsResults.TwilightSaw.Senders;
+
+public class EmailSender(EmailController emailController, MessageBuilder messageBuilder) : BackgroundService
+{
+    protected override async Task ExecuteAsync(CancellationToken stopToken)
+    {
+        SetInterval(() =>
+        {
+            var message = "Hello, there is your fresh news about the recent basketball matches:\n\n" + messageBuilder.GetGameMessage() + "\n" + messageBuilder.GetGameStatisticsMessage();
+            emailController.SendEmail("Name", "blackpanterastudio@gmail.com", message, "Today's NBA Games");
+        }, new TimeSpan(24,0,0)).Wait();
+    }
+
+    static async Task SetInterval(Action action, TimeSpan timeout)
+    {
+        await Task.Delay(timeout).ConfigureAwait(false);
```

Comment:
🟡 **Delay Before Running**

❓ Is it supposed to work like this? i.e., app start, wait a day, and then send? Why not: app start, send email, wait a day, repeat...